### ISSUE TITLE
 Product Details Block > Reviews > wrong order (Date Asc)

### ISSUE THREAD

**Issue by Marc-pi** [woothemes/woocommerce#42248]
## Describe the bug

The single product FSE template contains the Product Details blocks.
It is a massive block with tabs. One of them is the Review Tab.
The Review Tab lists all reviews for a given product, but those are sorted A (increasing) per date.
So the older date is first.

## To reproduce

1. Go to a single product that has several reviews
2. Observe the way the reviews are sorted
3. The oldest is first, which is wrong

## Expected behavior

By default Sort by Date D (the soonest first) : it's better for the user, and for SEO purpose
it is especially anoying when you have long life products (that lasts several years, like accommodations or services).
Fresh content must be first.

## Screenshots

![image](https://github.com/woocommerce/woocommerce-blocks/assets/919332/926dede6-d7ec-4d52-b522-a6a1c761578e)

## Environment
- TT3 theme
- WPP 6.3
- Gutenberg 16.4.0
- WooCommerce 8.0.2
- Woo Blocks 10.9.

**Comment by kmanijak** [comment]
Hi @Marc-pi!

I could reproduce it.

The block is consistent with the behavior of Classic Template as in fact, the Product Detail block uses the same functionality from WooCommerce Core underneath. Here's the piece of ([code](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/templates/single-product-reviews.php#L44)) responsible for getting the reviews.

In the meantime, you can influence it by reverting the order of reviews with the following code added to your `functions.php`:

```
add_filter( 'woocommerce_product_review_list_args', 'newest_reviews_first' );
function newest_reviews_first($args) {
    $args['reverse_top_level'] = true;
    return $args;
}
```

from [Stack Exchange](https://wordpress.stackexchange.com/a/133668).

Please let me know if that helps 🙌

**Comment by Marc-pi** [comment]
yep, but would be better to change the default order, which is more natural (think about social feeds : they are all sorted DESC)
i will bring more confidence for new incomers. (i did not check how shopify manages this by default)

**Comment by gigitux** [comment]
Closing this issue because it is correct that this block behavior aligns with the Classic Template. However, this scenario should be considered for the Blockified Product Details block. cc @louwie17 @imanish003
